### ISSUE TITLE
Clarify JXL read/write support

### ISSUE THREAD

**Issue by louie-github** [exiftool/exiftool#245]
Hello! On the README, JXL support is still listed as read-only:
```
         File Types
         ------------+-------------+-------------+-------------+------------
...
         AA    r     | DVR-MS r    | JXL   r     | OGV   r     | SEQ   r
...
```

However, I believe writing is supported as long as IO::Compress::Brotli is installed, and I even have to pass `-z` to compress it since I can write uncompressed metadata just fine too:
```
  Warning = Wrapped JXL codestream in ISO BMFF container
  Editing tags in: Exif ExifIFD File IFD0 JFIF JP2 MIE-Image MakerNotes Photoshop XMP xml
  Creating EXIF
  Rewriting IFD0
  Writing Brotli-compressed EXIF
  Creating XMP
  Writing Brotli-compressed XMP
```

I do not know if this is intentional or not, but I would like to request that the documentation be edited to reflect JXL r/w support, if that is really the case.

**Comment by boardhead** [comment]
Thanks for this report.  This was an oversight that will be fixed in version 12.78

- Phil

**Comment by boardhead** [comment]
I checked the version history, and JXL write support has been official since Apr. 22, 2021 - Version 12.25  :P

- Phil
